src/dodal/devices/pressure_jump_cell.py

```diff
@@ -200,25 +200,27 @@ def __init__(
         super().__init__(name)
 
 
-class PressureJumpCellController(HasName):
+class PressureJumpCellController(StandardReadable):
+    """
+    Top-level control for a fixed pressure or pressure jumps.
+    """
+
     def __init__(self, prefix: str, name: str = "") -> None:
-        self.stop = epics_signal_rw(StopState, f"{prefix}STOP")
+        with self.add_children_as_readables():
+            self.stop = epics_signal_rw(StopState, f"{prefix}STOP")
```

Comment:
Must: I know you didn't introduce this but having a signal called `stop` is bad as devices already have a `stop` function and this will confuse things. I have made https://github.com/bluesky/ophyd-async/issues/1010 to make this more obvious